clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/configuration/CamundaClientAllAutoConfiguration.java

```diff
@@ -90,15 +90,18 @@ public JobWorkerManager jobWorkerManager(
       final CommandExceptionHandlingStrategy commandExceptionHandlingStrategy,
       final MetricsRecorder metricsRecorder,
       final ParameterResolverStrategy parameterResolverStrategy,
-      final ResultProcessorStrategy resultProcessorStrategy) {
+      final ResultProcessorStrategy resultProcessorStrategy,
+      final BackoffSupplier backoffSupplier) {
     return new JobWorkerManager(
         commandExceptionHandlingStrategy,
         metricsRecorder,
         parameterResolverStrategy,
-        resultProcessorStrategy);
+        resultProcessorStrategy,
+        backoffSupplier);
   }
 
   @Bean
+  @ConditionalOnMissingBean
   public BackoffSupplier backoffSupplier() {
     return new ExponentialBackoffBuilderImpl()
```

Comment:
@jonathanlukas How do you feel about making these values the same as we have in the Java Client for consistency reasons?

```java
return BackoffSupplier.newBackoffBuilder().build();
```